### ISSUE TITLE
Crash on attribute and line continuation with no data

### ISSUE THREAD

**Issue by zelivans** [asciidoctor/asciidoctor#2880]
As promised another bug found with the fuzzer.

## Raw crash
```
/home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/parser.rb:2131:in `process_attribute_entry': asciidoctor: FAILED: <stdin>: Failed to load AsciiDoc document - undefined method `lstrip' for nil:NilClass (NoMethodError)
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/parser.rb:2107:in `parse_block_metadata_line'
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/parser.rb:2038:in `parse_block_metadata_lines'
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/parser.rb:123:in `parse_document_header'
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/parser.rb:93:in `parse'
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor/document.rb:565:in `parse'
	from /home/ariel/afl-kisaten/private/sandbox/asciidoctor/asciidoctor/lib/asciidoctor.rb:1360:in `load'
	from tst.rb:4:in `<main>'
```

## Explanation
Asciidoctor crashes with an uncaught `NoMethodError` in the `process_attribute_entry`. This happens when the it finds `LINE_CONTINUATION` or `LINE_CONTINUATION_LEGACY` in the end of a value but there is no further data so `reader.peek_line` returns nil, so the `lstrip` call fails.

Reproduce with the following input:

```
:t: t +
```

This is a minimized test case but it can of course work with any attribute and any line continuation symbol. In my PR I made a simple change to prevent the `lstrip` call on nil. Perhaps add a test case to check against this issue if you want.

**Comment by mojavelinux** [comment]
This is an incredible catch. I can't believe we've gotten this far and never noticed this before.

**Comment by mojavelinux** [comment]
Merged as 6e95520ed12617a13fb8d67db74daebaf653e3f5.
